public/script.js

```diff
@@ -34,16 +44,27 @@ async function convertTemplToGo() {
   let templCode = editor.getValue().trim();
 
   let goCode = window.ConvertTemplToGo(templCode);
-
+  
+  if (goCode.error != undefined) {
+    editor.setValue(goCode.code)
+    editor.getSession().setAnnotations([{
+      row: goCode.line,
+      column: 0,
+      text: goCode.error,
+      type: "error" // also "warning" and "info"
+    }])
+    return;
+  }
+  editor.setValue(templCode)
   fetch("https://play.golang.org/compile", {
     method: "POST",
-    body: JSON.stringify({ version: 2, body: goCode }),
+    body: JSON.stringify({ version: 2, body: goCode.result }),
   })
     .then((response) => response.json())
     .then((data) => {
-      // console.log("🚀 ~ .then ~ data:", data)
+      console.log("🚀 ~ .then ~ data:", data)
```

Comment:
Assume this and the other couple of console logs in here were just for debugging, is it worth removing them at this point?